scripts/gitlint_rules/rules.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from gitlint.rules import CommitRule, RuleViolation
+
+# class SignedOffBy(CommitRule):
```

Comment:
Add "Example gitlint configuration" as title - so that no one will go and delete